internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -1034,6 +1034,22 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 				log.Fatal(err)
 			}
 			jsonStructData = []interface{}{jsonPayloadInterface}
+		case "cloudflare_web_analytics_site":
+			jsonPayload, _, err := api.ListWebAnalyticsSites(context.Background(), cloudflare.AccountIdentifier(accountID), cloudflare.ListWebAnalyticsSitesParams{})
+			if err != nil {
+				log.Fatal(err)
+			}
+			resourceCount = len(jsonPayload)
+			m, _ := json.Marshal(jsonPayload)
+			err = json.Unmarshal(m, &jsonStructData)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			for i := 0; i < resourceCount; i++ {
+				jsonStructData[i].(map[string]interface{})["zone_tag"] = jsonStructData[i].(map[string]interface{})["ruleset"].(map[string]interface{})["zone_tag"]
```

Comment:
updated this to include the remapping as `site_tag` doesn't exist at the top level in the API response but it does in the resource.